test/python/common.py

```diff
@@ -249,19 +249,27 @@ def _(*args, **kwargs):
         try:
             import Qconfig
             QE_TOKEN = Qconfig.APItoken
-            QE_URL = Qconfig.config["url"]
+            QE_URL = Qconfig.config['url']
+            QE_HUB = Qconfig.hub if hasattr(Qconfig, 'hub') else None
+            QE_GROUP = Qconfig.group if hasattr(Qconfig, 'group') else None
+            QE_PROJECT = Qconfig.project if hasattr(Qconfig, 'project') else None
```

Comment:
Can you double check that this is consistent with the current `Qconfig.py` convention? The .sample expects the file to have a structure such as:
```
config = {
    'url': 'https://quantumexperience.ng.bluemix.net/api',
    'hub': 'hub',
    ...
}
```
and these lines seem to expect to have the variables defined outside the `config` dict.

Reply:
yes, thanks